### DISCUSSION TITLE
Zapier - Errore: Il totale dei pagamenti non corrisponde al totale da pagare.

### DISCUSSION THREAD

**Discussion by Claudiodic** [fattureincloud/api#469] [Q&A] [answered]
Ciao,
sto impazzendo da qualche giorno per inviare da Zoho CRM a Fatture in Cloud le fatture via Zapier.
Ho mappato tutti i campi seguendo la guida API di FiC ma mi da sempre l'errore in oggetto.
Si tratta di fattura con 2 righe, entrambe con iva al 22%.
nella list dei prodotti, inserisco correttamente l'id prodotto e l'id IVA (VAT) che nel mio caso è 0.
però mi da l'errore in oggetto.

ho messo false a "use gross prices" perchè essendo fatture sanitarie devo far vedere l'applicazione dell'IVA.

mi potete aiutare?
grazie

**Comment by Claudiodic** [comment]
@emajo pls help!

**Comment by Claudiodic** [comment]
Ciao,
non so come ma sono riuscito a risolvere il problema che la somma non faceva il totale (come diceva Totò)

ma ho un altro problema, nonostante su zapier io metta il Issued document item product id - [data.items_list[]product_id], quando viene generata la fattura su Fatture in Cloud quel campo non è popolato.

c'è un modo per risolvere?
grazie

**Reply by emajo** [reply]
Ciao @Claudiodic, il product id corrisponde a in id di un prodotto presente nell'account Fatture in Cloud?

**Reply by Claudiodic** [reply]
Ciao e grazie. 
si! È un product id esistente. 
Ho provato a metterlo singolo e in lista (separato da virgola) nel caso di fattura con 2 righe di prodotto ma non funziona. 

Product code lo lascio vuoto e ovviamente non lo popola (penso che se lo dovrebbe chiamare dall’ID)

Mentre da zapier compilo il nome e la descrizione e me li popola correttamente su fatture in cloud.

**Reply by emajo** [reply]
Si corretto, dovrebbe collegarsi con il product_id, per capire la motivazione andando a guardare i log delle richieste ti chiedo di contattare il nostro supporto clienti dalla chat all'interno del sito.

Grazie,
Emanuele.

**Reply by valmoz** [reply]
Buongiorno @Claudiodic 
è riuscito a risolvere il problema?

Grazie
Mauro

**Comment by Claudiodic** [comment]
no

Il giorno mer 11 set 2024 alle ore 09:52 Mauro Valota <
***@***.***> ha scritto:

**Reply by valmoz** [reply]
Ok, provo ad indagare con il supporto clienti.

Grazie
Mauro

**Comment by Claudiodic** [comment]
grazie

Il giorno mer 11 set 2024 alle ore 09:59 Mauro Valota <
***@***.***> ha scritto:

**Reply by valmoz** [reply]
@Claudiodic stiamo facendo fatica ad individuare la sua richiesta al supporto clienti, le consiglierei di rispondere al supporto aggiungendo il link a questa discussione in modo tale da poter unificare le due richieste (quella sulla community e quella su supporto clienti).

Grazie
Mauro

**Reply by valmoz** [reply, marked as answer]
Riporto la risposta fornita dal supporto (devo censurare alcune parti essendo questa una chat pubblica, può trovare la risposta completa su supporto clienti):

```
Buongiorno, ho recuperato le varie conversazione avvenute sia tramite la chat di assistenza sia tramite la Community di GitHub.

Come le indicava il collega analizzando la fattura di test n. ... del 03/09/2024 (cliente ...), che ora risulta nel cestino, vediamo che il product id risulta essere correttamente presente.

Le posso gentilmente chiedere di indicarci, magari tramite l'invio di uno screenshot, dove è di preciso che non vede questo product id indicato?

Le chiederei, se possibile, di ripristinare il documento dal cestino o comunque di lasciarci analizzare un documento che presente questo problema ma che non sia presente nel cestino.

Se quindi dovesse avere necessità di ricreare un documento verifichi se il problema si presenta e, se così fosse, ci avvisi subito, prima di cancellare il documento.

La ringrazio e resto a disposizione
```

Grazie
Mauro
